components/shared/forms/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useId } from "react";
+import styles from "./Checkbox.module.css";
+
+interface CheckboxProps {
+  isChecked: boolean;
+  setIsChecked: (event: any) => void;
+  label: string;
+}
+
+const Checkbox = ({ isChecked, setIsChecked, label }: CheckboxProps) => {
```

Comment:
Hi Alex, may I know why the Id attribute is necessary here? Let me know :D

Reply:
Hello Rashad,
I just put the Id attribute just to say which input element it belongs to. So that when the input element is clicked, it'll change the state of the checkbox.

Reply:
Okay Alex, got it!